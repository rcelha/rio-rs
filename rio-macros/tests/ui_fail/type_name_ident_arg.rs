use rio_macros::*;

#[derive(TypeName)]
#[type_name]
struct Test {
    pub a: u32,
}

fn main() {}
