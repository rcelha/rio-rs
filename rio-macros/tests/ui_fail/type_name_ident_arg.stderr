error: proc-macro derive panicked
 --> tests/ui_fail/type_name_ident_arg.rs:3:10
  |
3 | #[derive(TypeName)]
  |          ^^^^^^^^
  |
  = help: message: Expected "[type_name = "..."]"
